hikari/impl/entity_factory.py

```diff
@@ -1580,8 +1590,14 @@ def deserialize_partial_message(self, payload: data_binding.JSONObject) -> messa
             message_reference=message_reference,
             flags=message_models.MessageFlag(payload["flags"]) if "flags" in payload else undefined.UNDEFINED,
             nonce=payload["nonce"] if "nonce" in payload else undefined.UNDEFINED,
+            # We initialize these next.
+            mentions=NotImplemented,
         )
 
+        message.mentions = self._deserialize_mentions(payload, message)
```

Comment:
How would we tell the difference between the mentions being changed to no mentions vs the mentions not being changed (left unset and therefor not provided in the payload for message updates like embed specific ones)

Reply:
Forgot to implement, will change

Reply:
Implemented :+1: